runtime/doc/api.txt

```diff
@@ -3108,6 +3108,195 @@ nvim_tabpage_set_var({tabpage}, {name}, {value})
                     {value}    Variable value
 
 
+==============================================================================
+Autocmd Functions                                                *api-autocmd*
+
+nvim_create_augroup({*opts})                           *nvim_create_augroup()*
+                Create or get an autocommand group. See |autocmd-groups|.
+
+                To get an existing augroup id, do: >
+                    local id = vim.api.nvim_create_augroup({
+                      name = "MyGroup",
+                      clear = false
+                    });
+<
+
+                Parameters: ~
+                    {opts}  Dictionary Parameters
+                            • name: (string) The name of the augroup
+                            • clear: (bool), defaults to true. Whether to
+                              clear existing command or not for the group (if
+                              it exists). See |autocmd-groups|.
+
+                Return: ~
+                    Integer The id of the created augroup.
+
+                See also: ~
+                    autocmd-groups
+
+nvim_create_autocmd({*opts})                           *nvim_create_autocmd()*
+                Define a new autocommand, See |autocmd|.
+
+                Example: >
+                    vim.api.nvim_create_autocmd({
+                      event = {"BufEnter", "BufWinEnter"},
+                      pattern = {"*.c", "*.h"},
+                      callback = function() print("This buffer enters") end, 
+                      once = false,
+                    })
+<
+
+                Example values for patterns: >
+                  pattern = "*.py,*.pyi"
+                  pattern = "*.py"
+                  pattern = {"*.py"}
+                  pattern = { "*.py", "*.pyi" }
+                  -- NOT supported
+                  pattern = {"*.py,*.pyi"}
+<
+
+                Examples values for events: >
+                  event = "FileType,CursorHold"
+                  event = "BufPreWrite"
+                  event = {"BufPostWrite"}
+                  event = {"CursorHold", "BufPreWrite", "BufPostWrite"}
+<
+
+                NOTE: Requires one of pattern or buffer, requires one of
+                callback or command,
```

Comment:
Hmm, that belongs to the parameters list, and isn't fully reflected there. I'll have to think how to best fold that in.